website/static/downloads/swagger.yaml

```diff
@@ -2168,122 +2168,110 @@ components:
-        * 0x414E (16718 decimal) - Register Namespace Transaction.
-        * 0x424E (16974 decimal) - Address Alias Transaction.
-        * 0x434E (17230 decimal) - Mosaic Alias Transaction.
-        * 0x4154 (16724 decimal) - Transfer Transaction.
-        * 0x4155 (16725 decimal) - Modify Multisig Account Transaction.
-        * 0x4141 (16705 decimal) - Aggregate Complete Transaction.
-        * 0x4241 (16961 decimal) - Aggregate Bonded Transaction.
-        * 0x4148 (16712 decimal) - Hash Lock Transaction.
         * 0x4150 (16720 decimal) - Account Properties Address Transaction.
         * 0x4250 (16976 decimal) - Account Properties Mosaic Transaction.
-        * 0x4350 (17232 decimal) - Account Properties Entity Type Transaction.
-        * 0x4152 (16722 decimal) - Secret Lock Transaction.
-        * 0x4252 (16978 decimal) - Secret Proof Transaction.
-        * 0x414C (16716 decimal) - Account Link Transaction.
-        * 0x8043 (32835 decimal) - Nemesis Block Transaction.
-        * 0x8143 (33091 decimal) - Regular Block Transaction.
+        * 0x4350 (17232 decimal) - Account Properties Transaction.
+        * 0x424E (16974 decimal) - Address Alias Transaction.
+        * 0x4241 (16961 decimal) - Aggregate Bonded Transaction.
+        * 0x4141 (16705 decimal) - Aggregate Complete Transaction.
         * 0x415D (16733 decimal) - Add Exchange Offer Transaction.
+        * 0x4161 (16737 decimal) - Add Harvester Transaction.
         * 0x425D (16989 decimal) - Exchange Offer Transaction.
         * 0x435D (17245 decimal) - Remove Exchange Offer Transaction.
-        * 0x4157 (16727 decimal) - Modify Contract Transaction.
-        * 0x415A (16730 decimal) - Prepare Drive Transaction.
-        * 0x425A (16986 decimal) - Join To Drive Transaction.
-        * 0x435A (17242 decimal) - Drive File System Transaction.
-        * 0x445A (17498 decimal) - Files Deposit Transaction.
-        * 0x455A (17754 decimal) - End Drive Transaction.
-        * 0x465A (18010 decimal) - Drive Files Reward Transaction.
-        * 0x475A (18266 decimal) - Start Drive Verification Transaction.
-        * 0x485A (18522 decimal) - End Drive Verification Transaction.
-        * 0x4161 (16737 decimal) - Add Harvester Entity Type Transaction.
-        * 0x4261 (16993 decimal) - Remove Harvester Entity Type Transaction.
+        * 0x4261 (16993 decimal) - Remove Harvester Transaction.
+        * 0x8143 (33091 decimal) - Regular Block.
+        * 0x8043 (32835 decimal) - Nemesis Block.
+        * 0x4158 (16728 decimal) - Blockchain Upgrade Transaction.
+        * 0x4159 (16729 decimal) - Network Config Transaction.
+        * 0x414C (16716 decimal) - Account Link Transaction.
+        * 0x4148 (16712 decimal) - Hash Lock Transaction.
+        * 0x413D (16701 decimal) - Metadata Address Transaction.
+        * 0x423D (16957 decimal) - Metadata Mosaic Transaction.
+        * 0x433D (17213 decimal) - Metadata Namespace Transaction.
```

Comment:
just recall that are replaced by 
```go
        * 0x413f (16703 decimal) - Account Metadata Transaction.
        * 0x423f (16959 decimal) - Mosaic Metadata Transaction.
        * 0x433f (17215 decimal) - Namespace Metadata Transaction.
```

So can be removed

Reply:
alright. have removed them in the latest commit